src/runway/commands/runway/whichenv.py

```diff
@@ -31,5 +31,5 @@ def execute(self):
         logging.getLogger('runway').setLevel(logging.ERROR)  # suppress warnings
         print(get_env(
             self.env_root,
-            self.runway_config.get('ignore_git_branch', False)
+            self.runway_config.get('ignore-git-branch', False)
```

Comment:
With the `Config` class, all possible options have a value so we don't have to use `.get` here and instead use the attribute directly.

```suggestion
            self.runway_config.ignore_git_branch),
```

Also, since this is `.get`ing a class attribute, it does not have hyphens and will need to be reverted to underscores.